css-forms-1/Overview.bs

```diff
@@ -843,8 +843,14 @@ ISSUE: Refine meter, progress, switch and range input styling.
 
 ```css
 input:is([type=checkbox]:not([switch]), [type=radio]) {
-    width: 1em;
-    height: 1em;
+    /* These size rules ensure accessibility by following WCAG rules:
+    * https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html */
+    block-size: max(1em, 24px);
```

Comment:
I think this should probably get an aspect-ratio: 1/1; by default too so you don't end up with accidentally rectangular checkboxes.

In my demo implementation the checkmark can end up stretching the checkbox weirdly without it.